### DISCUSSION TITLE
React Vite PostCSS unable use custom classes with `@apply` within `style.module.css`

### DISCUSSION THREAD

**Discussion by AndrewSunshiny** [tailwindlabs/tailwindcss#10683] [Help]
Hi everyone, I have an issue

I cannot use custom css classes declared with `@layer` directive within component's `style.module.css` files

I have custom css class `.grid-container`

// index.css
![image](https://user-images.githubusercontent.com/50839774/221388569-73a27dff-c72f-4ffe-905d-0df93a42580e.png)

// Button.module.scss
![image](https://user-images.githubusercontent.com/50839774/221388600-7bee2737-2262-4f94-8f76-24eae8ebf903.png)

And after rendering Vite returns this message:
![image](https://user-images.githubusercontent.com/50839774/221388695-fe106a40-2740-4a3a-b6dc-a3dd0a339ee1.png)

**Comment by adamwathan** [comment]
Hey! Relevant docs for this are here: https://tailwindcss.com/docs/functions-and-directives#using-apply-with-per-component-css
